src/meta_data.rs

```diff
@@ -138,17 +138,17 @@ impl MetaData {
     }
 
     /// Build `gRPC` client to `DatenLord` worker
-    pub fn build_worker_client(&self, node_id: &str) -> WorkerClient {
+    pub fn build_worker_client(node_id: &str, worker_port: u16) -> WorkerClient {
         // let env = Arc::new(EnvBuilder::new().build());
```

Comment:
Could we just pass DatenLordNode instead of node_id and worker_port?
As in PR https://github.com/datenlord/csi/pull/22, we need support ip address.

Reply:
Done.